README.md

```diff
@@ -49,10 +49,147 @@ $ yarn lint
+- this hash can be only seen with use of `DEBUG=superface:testing:hash*`
+
+### Test with live traffic
+
+```shell
+$ yarn test:record
+```
+
+With this command, live traffic is enabled and therefore maps send requests to the live providers. This command also records this request and response. There are multiple scenarios what can happen:
+
+- use-case does not have recording → records traffic and saves as default one
+- use-case already have recording → records new traffic and compares it to the old one →
+    - if it matches → do not save recording
+    - if it doesn’t match → save new recording next to old one with suffix `-new`
+
+### Test with new mocked traffic
+
+```shell
+$ yarn test:with_new_traffic
+```
+
+This command run tests without live traffic, similar as `yarn test`, but uses new recordings instead of default ones (if present). New recordings are always newly recorded traffic that does not match the default old one and are located next to the old one with suffix `-new`.
+
+### Test and replace old recording with new one
+
+**⚠️ This command should be run only when you’re sure that tests are passing with new recordings → `yarn test:with_new_traffic`**
+
+```shell
+$ yarn test:update_traffic
+```
+
+This commands run tests with mocked traffic, but also replaces old recording with new one (if present) before loading recordings - this means that tests are run with mocked new traffic already.
+
+### Test with live traffic with development reporting
+
+```shell
+$ yarn test:record:dev
+```
+
+This command run tests with live traffic and record it, similar as `yarn test:record`. The script implementation looks like this:
+
+```shell
+$ TEST_ENV='dev' yarn test:record --group=live/safe
+```
+
+`TEST_ENV` sets development environment for reporter - to use local reporter that just logs the provider changes into console
+
+`--group=live/safe` filters what tests to run, thanks to [jest-runner-groups](https://www.npmjs.com/package/jest-runner-groups) 
+
+### Test with live traffic with production reporting
+
+```shell
+$ yarn test:record:prod
+```
+
+This command run tests with live traffic and record it, similar as `yarn test:record` or `yarn test:record:dev`. Only difference between `test:record:dev` is that it uses production reporter that reports provider changes to the public slack channel.
+
+Slack channel can be changed with env variable `PROD_REPORTING_DESTINATION`. It expects channel id.
+
+### Test and update snapshots
+
+Since lot of tests written in station uses jest snapshots for comparing values from map, you might need to update snapshots to modify/add/remove some of them. You can do this simply by adding option `-u` or `--updateSnapshot`. For example:
+
+```shell
+$ yarn test -u
+
+$ yarn test:record --updateSnapshot
+```
+
+---
+
+### Overall filtering of tests
+
+In each of command described above, you can use argument to specify integration, for example:
```

Comment:
This doesn't explain what I should enter here. Is it a directory path or use case identifier (with scope and profile an provider tacked on)?

Also, can I use a glob (like `chat/*/maps/slack`)?